Makefile

```diff
@@ -3,12 +3,12 @@ all: clean ocs-operator ocs-must-gather
 .PHONY: ocs-operator
 ocs-operator:
 	@echo "Building ocs-operator image"
-	operator-sdk build quay.io/openshift/ocs-operator
+	operator-sdk build quay.io/ocs-dev/ocs-operator
 
 .PHONY: ocs-must-gather
 ocs-must-gather:
 	@echo "Building ocs-must-gather image"
-	docker build -f must-gather/Dockerfile -t quay.io/openshift/ocs-must-gather:latest must-gather/
+	docker build -f must-gather/Dockerfile -t quay.io/ocs-dev/must-gather:latest must-gather/
```

Comment:
Typo: it must be `quay.io/ocs-dev/ocs-must-gather:...`

Reply:
(and also: separate PR)

Reply:
dropped the makefile as @kshlm has raised a PR for this. So marking this comment as resolved.